psmdb/percona-server-for-mongodb-4.2-template.yml

```diff
@@ -64,22 +64,22 @@
         set -x
 
         if [ -f /usr/bin/apt-get ]; then
-            sudo apt-get -y install wget
+            sudo apt-get -y install wget wiredtiger
```

Comment:
why?

Reply:
pls remove for now, I'll add it if required